sqlitecluster/SQLite.cpp

```diff
@@ -34,7 +34,7 @@ string SQLite::initializeFilename(const string& filename) {
     }
 }
 
-SQLite::SharedData& SQLite::initializeSharedData(sqlite3* db, const string& filename, const vector<string>& journalNames) {
+SQLite::SharedData& SQLite::initializeSharedData(sqlite3* db, const string& filename, const vector<string>& journalNames, bool _wal2) {
```

Comment:
```suggestion
SQLite::SharedData& SQLite::initializeSharedData(sqlite3* db, const string& filename, const vector<string>& journalNames, bool enableWAL2) {
```

Since `_wal2` doesnt lend itself to be seen as `bool`